test/rastrigin.jl

```diff
@@ -5,7 +5,7 @@
             @test ≈(result, zeros(N), atol=tol)
             @test ≈(fitness,0.0, atol=tol)
         else
-            @warn("Found local minimum!!!")
+            # @warn("Found local minimum!!!")
```

Comment:
Why did you comment this line?

Reply:
I don't think this should be a warning if it is a valid outcome of a test. In my experience, it was just polluting stdout.

Reply:
Oh! I just run some test and saw it. Never mind then.